node/src/components/consensus/highway_core/state.rs

```diff
@@ -705,6 +769,29 @@ impl<C: Context> State<C> {
     pub(crate) fn is_empty(&self) -> bool {
         self.units.is_empty()
     }
+
+    /// Returns the panorama of the confirmation for the leader unit `vhash`.
+    pub(crate) fn confirmation_panorama(
+        &self,
+        vidx: ValidatorIndex,
```

Comment:
Maybe add a comment that this is _our_ index, or:
```suggestion
        own_vidx: ValidatorIndex,
```

Reply:
Done in https://github.com/CasperLabs/casper-node/pull/550/commits/dbc3dbbf6d706de6052bd675f2fe6b51088e81dd.